pkg/loadbalancer/tasks.go

```diff
@@ -182,20 +186,50 @@ func (g *GenerateKubevipManifest) Execute(runtime connector.Runtime) error {
 	if !ok {
 		return errors.New("get interface failed")
 	}
-
-	templateAction := action.Template{
-		Template: templates.KubevipManifest,
-		Dst:      filepath.Join(common.KubeManifestDir, templates.KubevipManifest.Name()),
-		Data: util.Data{
-			"VipInterface": interfaceName,
-			"KubeVip":      g.KubeConf.Cluster.ControlPlaneEndpoint.Address,
-			"KubevipImage": images.GetImage(runtime, g.KubeConf, "kubevip").ImageName(),
-		},
-	}
-
-	templateAction.Init(nil, nil)
-	if err := templateAction.Execute(runtime); err != nil {
-		return err
+	if g.KubeConf.Cluster.ControlPlaneEndpoint.KubeVip.Mode == "BGP" {
+		hosts := runtime.GetHostsByRole(common.Master)
+		BGPPeers := ""
+		for _, value := range hosts {
+			address := value.GetAddress()
+			if address != host.GetAddress() {
+				if BGPPeers == "" {
+					BGPPeers = fmt.Sprintf("%s:65000::false", address)
+				} else {
+					BGPPeers = fmt.Sprintf("%s,%s:65000::false", BGPPeers, address)
+				}
+			}
+		}
+		templateAction := action.Template{
+			Template: templates.KubevipManifest,
+			Dst:      filepath.Join(common.KubeManifestDir, templates.KubevipManifest.Name()),
+			Data: util.Data{
+				"BGPMode":      true,
```

Comment:
What do you think of adding a new template variable rather than using this `BGPMode` to select different string in the golang template engine?